### DISCUSSION TITLE
Setting Diode Parameters

### DISCUSSION THREAD

**Discussion by Abbas111977** [SpiceSharp/SpiceSharp#194] [Q&A] [answered]
Dears,
I would like to know how to change certain parameter ( Breakdown voltage )in diode model after creating its object.
Best Regards

**Comment by marcin-golebiowski** [comment, marked as answer]
Please take a look at following example:

https://github.com/SpiceSharp/SpiceSharp/blob/master/SpiceSharpTest/Examples/ChangingParameter/Changing.cs

If it doesn't help you, please let us know.

**Reply by Abbas111977** [reply]
Thank you very much, I shall try it 👍
